extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -780,6 +840,37 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 		kernelSpec.argv = kernelSpec.argv?.map(arg => arg.replace('{ADS_PYTHONDIR}', this._pythonInstallationPath));
 		await fs.writeFile(kernelPath, JSON.stringify(kernelSpec, undefined, '\t'));
 	}
+
+	private async createInstallPipPackagesHelpNotebook(userInstalledPipPackages: PythonPkgDetails[]): Promise<void> {
+		let packagesList: string[] = userInstalledPipPackages.map(pkg => { return pkg.name; });
+		let installPackagesCode = `import sys\n!{sys.executable} -m pip install --user ${packagesList.join(' ')}`;
+		let initialContent: azdata.nb.INotebookContents = {
+			cells: [{
+				cell_type: 'markdown',
+				source: ['# Install Pip Packages\n\nThis notebook will help you reinstall the pip packages you were previously using so that they can be used with Python 3.8.\n\n**Note:** Some packages may have a dependency on Python 3.6 and will not work with Python 3.8.\n\nRun the following code cell after Python 3.8 installation is complete.'],
+			}, {
+				cell_type: 'code',
+				source: [installPackagesCode],
+			}],
+			metadata: {
+				kernelspec: {
+					name: 'python3',
+					language: 'python3',
+					display_name: 'Python 3'
+				},
+				language_info: {
+					name: 'python3'
+				}
+			},
+			nbformat: 4,
+			nbformat_minor: 5
+		};
+
+		await vscode.commands.executeCommand('_notebook.command.new', {
```

Comment:
Instead of calling the command like this it would be clearer to just have the functions in the Utils class just be module-level functions instead of being part of a class. There's no reason for the class to exist and it just makes things harder to follow. 

(this should probably be done in a separate PR though to keep this one focused on the changes being made)